crates/typst/src/visualize/image/mod.rs

```diff
@@ -32,13 +32,12 @@ use crate::utils::LazyHash;
 use crate::visualize::Path;
 use crate::World;
 
-/// A raster or vector graphic.
+/// ラスターまたはベクター画像
```

Comment:
原文ではここは段落要素`<p>`なので、句点で終わるべきだと思います。

```suggestion
/// ラスターまたはベクター画像。
```

Reply:
タイトル直後の部分ですね。
このご指摘は大変ありがたいです。句点は私も判断が悩んだところなので、ご相談させてください。

[JTFスタイルガイド](https://www.jtf.jp/tips/styleguide)によると、「見出しや図表のキャプションについては、体言止めを使い句点は付けない」としていますが、「それ以外の部分で体言止めにしてはいけない」などの記載はありません。例えばParametersの`width`や`height`も`<p>`になっていると思います。タイトルの直後とパラメーターの説明についてはある種キャプション的な扱いをしてしまっても自然かと思い、「変に文章として成立させるよりかは素直に体言止めで訳して句点を付けない」という対応を取りました。JTFスタイルガイドのP. 10の図表のキャプションでの記載

> 通常、文末に句点（。）を付けませんが、複数の文章になる場合は句点を使用します。

に従い、今回でいうと`format`などは後ろに文が続くため体言止めですが句点を使用しております。

- タイトル直後の部分は他のページでも自然に訳すとほとんどが体言止めとなると思います。全体に関わることなので @mkpoli さんのご感想も伺いたいです。方針が決まれば #78 での対応になるかもしれませんが、翻訳手順の`### 表記`に追加したほうがよいかもしれませんね。

ご意見のほど、よろしくお願いいたします。

Reply:
それらは連動しているので、どちらかに決めるしかありません。
今回の例でいうとVisualizeのページのmod.rsがimageの定義を引いてきています。

https://github.com/typst-jp/typst-jp.github.io/blob/878a8b3b0bf6a5a12111468aaa02b5ff310aa1c9/crates/typst/src/visualize/mod.rs#L35-L50

統一されていればどちらでも問題が無さそうですが、迷うようであれば原文に近い体言止めを推します。

- 総括（Visualizeの方）は箇条書きとなるため体言止めとする方が良さそう。文章となっても統一されていれば問題は無さそう？
- 本文（imageの方）は後ろに文が続くため文章にする方が良さそう？必ず改段落がされるため、体言止めでも問題無さそう？

Reply:
なるほど、ではどちらか一方を決めておく必要がありますね。

そうすると、本文の方で後ろに文章が続いているのに、句点が最初だけない、というのは変なので、

やはり句点は必要そうです。「です」はあってもなくてもいい気がしますが、原文通りなら体言止めにするとなると

折衷案として「体言。」のように句点付きの体言止めがよろしいかと存じます。

Reply:
rustdoc中の文書は原文を尊重して句点を付けて訳し、Webサイト上で箇条書きとして扱われる部分については静的サイトの生成時に句点を取る処理を追加すると良いかもしれません。

Reply:
@mkpoli @3w36zj6 
ご意見いただき誠にありがとうございました。大変参考になりました。
他の翻訳者に判断負荷をかけるのも避けたいため、ひとまず、コード上は体言止めも含めてすべてに句点をつけるという対応を取りました。
いかがでしょうか？